packages/react-scripts/template/.travis.yml

```diff
@@ -5,8 +5,14 @@ before_install:
   - echo -e "machine github.com\n  login $CI_USER_TOKEN" > ~/.netrc
   - echo "//code.lds.org/artifactory/api/npm/npm-fhd/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
   - echo "@fs:registry=https://code.lds.org/artifactory/api/npm/npm-fhd/" >> ~/.npmrc
-  - npm i -g npm@6.8.0
-script: npm run lint && npm run test
+  - npm i -g npm@latest
+script: npm run lint && npm run test -- --coverage
 branches:
   only:
-    - master
\ No newline at end of file
+    - master
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
+after_script:
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
I think this will harmlessly fail until they add the CC env vars, but if we're bothered by it, I guess we create a script/npm task to add this?

Reply:
fine with me

Reply:
It does not break builds, just logs an error in the Travis console:
```
0.21s$ ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
Error: you must supply a CC_TEST_REPORTER_ID ENV variable or pass it via the -r flag
```